app/models/movie.rb

```diff
@@ -2,4 +2,5 @@
 
 class Movie < ApplicationRecord
   has_many :screenings, dependent: :destroy
+  attribute :length, :string
```

Comment:
What would you like to achieve here?

Reply:
When I dont have it here and try to create new movie I get a deprecation warning and the length is put as nil. With this line it fixes the issue

Reply:
DEPRECATION WARNING: The behavior of the `:interval` type will be changing in Rails 6.2
to return an `ActiveSupport::Duration` object. If you'd like to keep
the old behavior, you can add this line to Movie model:

  attribute :length, :string